src/closures/hello_closure.rs

```diff
@@ -31,7 +43,10 @@ pub struct z_loaned_closure_hello_t {
     _0: [usize; 3],
 }
 
-decl_transmute_handle!(z_owned_closure_hello_t, z_loaned_closure_hello_t);
+decl_c_type!(
```

Comment:
`decl_c_type` and `as_rust_type_mut` but `as_loaned_ctype_ref` 
`as_loaned_c_type_ref` will be more consistent

Reply:
agree, fixed